administrator/language/en-GB/plg_content_vote.ini

```
@@ -9,7 +9,9 @@ PLG_VOTE_LABEL="Please Rate"
 PLG_VOTE_POSITION_LABEL="Position"
 PLG_VOTE_RATE="Rate"
 PLG_VOTE_STAR_ACTIVE="Star Active"
-PLG_VOTE_STAR_INACTIVE="Star Inactive"
+PLG_VOTE_STAR_ACTIVE_HALF="Star Half Active"
+PLG_VOTE_STAR_INACTIVE="Star Not Active"
+PLG_VOTE_STAR_RATING="Rating: %s out of 5 stars"
```

I don't think "stars" is necessary here. Stars are just visual representation which can be changed to anything really.

---

[ARIA: img role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img)
..or by using an _aria-label_ attribute, with optional alt attributes _for search engines_ or sighted users to be written to the page should an image fail..
So, for the sake of completeness.

---

> I don't think "stars" is necessary here

Removed (most) language strings.

---

I agree with @SharkyKZ 
```suggestion
PLG_VOTE_STAR_RATING="Rating: %s out of 5"
```

---

"Stars" deleted